src/lab_sim/objectives/run_sam2_onnx.xml

```diff
@@ -11,31 +11,77 @@
   >
     <Control ID="Sequence">
       <Action ID="ClearSnapshot" />
-      <Action ID="GetImage" topic_name="/wrist_camera/color" />
+      <Action
+        ID="GetImage"
+        topic_name="/wrist_camera/color"
+        timeout_sec="5.000000"
+        message_out="{image}"
+      />
       <Action
         ID="GetPointsFromUser"
         point_prompts="Select the object to be segmented;"
         point_names="Point1;"
         view_name="/wrist_camera/color"
+        pixel_coords="{pixel_coords}"
+      />
+      <Action
+        ID="GetMasks2DFromPointQuery"
+        image="{image}"
+        pixel_coords="{pixel_coords}"
+        masks2d="{masks2d}"
+        decoder_model_path="~/user_ws/src/example_behaviors/models/decoder.onnx"
```

Comment:
@mkhansenbot You could set the `model_package` port to example_behaviors and use the .onnx in install/, but that requires removing COLCON_IGNORE from that package so I just gave it the file location in src/